src/classes/UTIL_Permissions.cls

```diff
@@ -168,14 +168,14 @@ public with sharing class UTIL_Permissions {
     }
 
     /**
-    * @description Instance method which can be mocked
-    * Determines whether the running user has delete permissions on the given object
+    * @description Determines whether the running user has delete permissions on the given object
+    * This is an instance method which can be mocked.
     *
-    * @param sObjectType SObjectType to check
+    * @param sObjectName Name of the sObject to check
     * @return returns boolean to indicate whether the user has permission
     */
-    public Boolean canDelete(Schema.SObjectType sObjectType) {
-        return canDelete(sObjectType.getDescribe().getName(), false);
+    public Boolean canDeleteSObject(String sObjectName) {
```

Comment:
Can the new instance method take the object token as arg instead of a string?  Sorry if I missed this in our conversation earlier.  I don't think we should introduce any new string representations of object/field representations unless we really have to.